content/docs/command-reference/import-url.md

```diff
@@ -125,12 +125,13 @@ source.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
-- `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
-  the file or directory already exists, this can be used to skip the download.
-  The data hash is not calculated when this option is used, only the import
-  metadata is saved to the `.dvc` file. `dvc commit <out>.dvc` can be used if
-  the data hashes are needed in the `.dvc` file, and to save existing data to
-  the cache.
+- `--no-exec` - create the import `.dvc` file but don't download `url`. The No
+  hash values are calculated or saved to the `.dvc` file. Use `dvc update` to
+  finish the operation later, or `dvc commit` if the data has already been
+  downloaded manually.
+
+  > Note that there's no guarantee that the data source is valid until the
```

Comment:
can we embed this ... like keep three - four words and embed into the paragraph

Reply:
Embedded.